### ISSUE TITLE
[Feature Request] Delete Songs/Folders within the App

### ISSUE THREAD

**Issue by davidgermany** [zyrouge/symphony#245]
### Description

Other OPEN-SOURCE Music Players like have this feature, and I really miss it here.

I think the layout in this music player is FANTASTIC, but I just want to long press on some old Songs and delete them from my device so hard. 

If you would add this feature this would be the absolut best music player, and I would definitely buy you a coffee. I know that 1 Coffee does not pay for development, but I'm sure other people will appreciate that feature as well.

TL:DR
make it so that you can long press on a Song (also include multiselect, like you know, I long press one song and then I can just click the others) and then make it so that I can press a bin button to delete it, but before it actually deletes ask me again "are you sure?" just so that this couldn't accidentally happen.

This would make me very happy ♥️

**Comment by Werewolfke** [comment]
Yes! Also something I miss!
I download a few thousand songs at a time, listen to them and if I don't like one, I want to delete it while it's on the player, so somewhere press the 3 buttons, delete, and it deletes the song and starts playing the next song.
Now I have to go to my file system, go to the folder, and find it there... Its not that convenient 😅

Otherwise same, Love the app and its unique features that many players don't have, and the way it looks and handles music... Amazing! Just this tiny little feature 😅❤️

**Comment by Giorgio-Germani** [comment]
Yeah a delete button is something i also wish any player would come with. 

I think a great way would be to have a button with a trashcan icon on the very right of the screen, just next to play/pause, back and next button. then a confirmation popup would appear and you can confirm delete.

**Comment by pratikmmohite** [comment]
@zyrouge Is this feature picked up by someone or I like to try it out.

**Comment by zyrouge** [comment]
> [@zyrouge](https://github.com/zyrouge) Is this feature picked up by someone or I like to try it out.

I'd suggest you to wait till the next release before proceeding with any changes since there will be a major overhaul in code.
